mainwindow.ui

```diff
@@ -44,6 +44,9 @@
            <property name="text">
             <string>Choose directory</string>
            </property>
+           <property name="icon">
+            <iconset theme="document-open-folder"/>
```

Comment:
seems that the UI compiler doesn’t do fallbacks.

we’ll have to find a better way, likely using `setThemeSearchPaths`